gitlab/const.py

```diff
@@ -49,6 +83,20 @@
 SEARCH_SCOPE_BLOBS: str = "blobs"
 SEARCH_SCOPE_USERS: str = "users"
 
+
+# Search scopes
+# all scopes (global, group and  project)
+class SearchScope(Enum):
+    SEARCH_SCOPE_PROJECTS: str = "projects"
+    SEARCH_SCOPE_ISSUES: str = "issues"
+    SEARCH_SCOPE_MERGE_REQUESTS: str = "merge_requests"
+    SEARCH_SCOPE_MILESTONES: str = "milestones"
+    SEARCH_SCOPE_WIKI_BLOBS: str = "wiki_blobs"
+    SEARCH_SCOPE_COMMITS: str = "commits"
+    SEARCH_SCOPE_BLOBS: str = "blobs"
+    SEARCH_SCOPE_USERS: str = "users"
+
+
 # specific global scope
```

Comment:
What about these 2 below, should we introduce GlobalSearchScope and ProjectSearchScope, or just add them to the above? We should look at whether gitlab does anything with this, or only provides these as strings in API filters.

Reply:
good point, I added them.